scripts/bump-changelogs.sh

```diff
@@ -41,6 +41,10 @@ do
 done
 
 rm -rf $CHAP
-printf "\n!!!!!!\n%s %s\n!!!!!!\n" \
-  "WARNING! DO NOT BUMP THE VERSION NUMBER IN THE CABAL FILES" \
-  "(unless its dependencies were bumped)!"
+BUMPED=$(git diff --exit-code)
+if [[ $BUMPED ]]; then
+  printf "\n!!!!!!\n%s %s\n!!!!!!\n" \
+    "WARNING! DO NOT BUMP THE VERSION NUMBER IN THE CABAL FILES" \
+    "(unless its dependencies were bumped)!"
```

Comment:
I think "their" sounds better? 

```suggestion
    "(unless their dependencies were bumped)!"
```